optimize/self-managed/optimize-deployment/configuration/system-configuration-platform-7.md

```diff
@@ -10,7 +10,9 @@ to import data from. Each engine configuration should have a unique alias associ
 with it and represented by `${engineAlias}`.
 
 Note that each connected engine must have its respective history level set to `FULL` in order to see all available data
-in Optimize. Using any other history level will result in less data and/or functionality within Optimize.
+in Optimize. Using any other history level will result in less data and/or functionality within Optimize. Furthermore,
+history in a connected engine should be configured for long enough for Optimize to import it. If data is removed from an
```

Comment:
"long enough for Optimize to import it"

Can we be more specific, or does it depend on the installation and config?

Reply:
I wish we could 😢 
Unfortunately it depends on a lot of things so we tend to avoid being specific with configuration like this in our docs. Normally the aggressiveness with which users delete their engine data is determined more by their engine performance than Optimize downstream. I feel like the best we can realistically do is warn them of the effects this might have in cases where the import can't keep up for whatever reason

Reply:
Fair enough. Maybe one day 🙂